app/models/review_turnaround.rb

```diff
@@ -2,24 +2,24 @@
 #
 # Table name: review_turnarounds
 #
-#  id                :bigint           not null, primary key
-#  value             :integer
-#  created_at        :datetime         not null
-#  updated_at        :datetime         not null
-#  review_request_id :bigint           not null
+#  id              :bigint           not null, primary key
+#  value           :integer
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  pull_request_id :bigint
 #
 # Indexes
 #
-#  index_review_turnarounds_on_review_request_id  (review_request_id)
+#  index_review_turnarounds_on_pull_request_id  (pull_request_id)
 #
 # Foreign Keys
 #
-#  fk_rails_...  (review_request_id => review_requests.id)
+#  fk_rails_...  (pull_request_id => pull_requests.id)
 #
 
 class ReviewTurnaround < ApplicationRecord
-  belongs_to :review_request
+  belongs_to :pull_request, class_name: 'Events::PullRequest'
```

Comment:
If we associate with the PR, will we have problems if we want to have different metrics in the future? For example if we want to track `Time to first review`?